libs/ui/inputs/src/lib/navigation-button/navigation-button.component.html

```diff
@@ -1,5 +1,5 @@
 <button
-  class="group flex items-center justify-center p-1 bg-primary-opacity-25 text-white rounded"
+  class="group flex items-center justify-center p-1 bg-white opacity-80 box-shadow-xl shadow-[4.0px_8.0px_8.0px_rgba(0,0,0,0.38)] mix-blend-luminosity text-black rounded hover:opacity-100"
```

Comment:
It's better use `text-main` instead of `text-black` here

Also I don't think we should customize the box shadow, it will end up not being consistent with others I think? Although I agree that the box shadows in tailwind are quite faint. I would suggest doing what I did in my comment, which is have a separate element for the background color and only change the opacity of this element; the rest can stay at 100% opacity and thus the shadow will be stronger. But I may be wrong, to be confirmed with trials :)